src/pages/services/aem-cf-console-admin/api/action-bar/index.md

```diff
@@ -145,3 +145,62 @@ const guestConnection = await register({
 | Field    | Type                                                                        | Required | Description                                                                  |
 |----------|-----------------------------------------------------------------------------| ------ |------------------------------------------------------------------------------|
 | id       | `string`                                                                    | ✔️  | "id" is a "data-id" attribute of the button (please use browser developer console to find actual value). |
+
+## Examples:
+### Buttons conditional rendering
+
+In React, rendering occurs whenever the state or props of a component change. If the getButtons method is called within the rendering cycle of a component, it will be executed whenever that component is re-rendered due to state or prop changes.
+
+You can indeed use this method to dynamically return different sets of buttons based on the data that has changed on the page. For example:
```

Comment:
Why explain to 3rd party developers internal implementation?
I think you can have only this statement: "Content Fragment Admin invokes getButtons method each time a user changes content fragments selection. Therefore, you can dynamically return different sets of buttons based on the data that has changed on the page. For example:
Comment
"